languageserver/client/src/cadence.ts

```diff
@@ -0,0 +1,191 @@
+import * as monaco from "monaco-editor";
+
+export const CADENCE_LANGUAGE_ID = "cadence"
+
+interface CadenceMonarchLanguage extends monaco.languages.IMonarchLanguage {
+}
+
+export function configureCadence() {
+
+  monaco.languages.register({
+    id: CADENCE_LANGUAGE_ID,
+    extensions: [".cdc"],
+    aliases: ["CDC", "cdc"]
+  });
+
+  monaco.languages.setMonarchTokensProvider(CADENCE_LANGUAGE_ID, {
+
+    keywords: [
+      "if",
+      "else",
+      "return",
+      "continue",
+      "break",
+      "while",
+      "pre",
+      "post",
+      "prepare",
+      "execute",
+      "import",
+      "from",
+      "create",
+      "destroy",
+      "priv",
+      "pub",
+      "get",
+      "set",
+      "log",
+      "emit",
+      "event",
+      "init",
+      "struct",
+      "interface",
+      "fun",
+      "let",
+      "var",
+      "resource",
+      "access",
+      "all",
+      "contract",
+      "self",
+      "transaction"
+    ],
+
+    typeKeywords: [
+      "AnyStruct",
+      "AnyResource",
+      "Void",
+      "Never",
+      "String",
+      "Character",
+      "Bool",
+      "Self",
+      "Int8",
+      "Int16",
+      "Int32",
+      "Int64",
+      "Int128",
+      "Int256",
+      "UInt8",
+      "UInt16",
+      "UInt32",
+      "UInt64",
+      "UInt128",
+      "UInt256",
+      "Word8",
+      "Word16",
+      "Word32",
+      "Word64",
+      "Word128",
+      "Word256",
+      "Fix64",
+      "UFix64"
+    ],
+
+    operators: [
+      "<-",
+      "<=",
+      ">=",
+      "==",
+      "!=",
+      "+",
+      "-",
+      "*",
+      "/",
+      "%",
+      "&",
+      "!",
+      "&&",
+      "||",
+      "?",
+      "??",
+      ":",
+      "=",
+      "@"
+    ],
+
+    // we include these common regular expressions
+    symbols: /[=><!~?:&|+\-*\/\^%]+/,
```

Comment:
Might want to add `#` now for pragmas!

Reply:
Yeah, this was just copied over from the Playground and we needs to be updated

Reply:
Anything else to add?